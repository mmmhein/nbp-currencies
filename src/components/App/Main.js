import styled from 'styled-components';

const Main = styled.main`
  margin-top: 6rem;
`;

export default Main;
