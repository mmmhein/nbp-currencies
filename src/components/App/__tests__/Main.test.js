import React from 'react';
import { create } from 'react-test-renderer';

import Main from '../Main';

describe('COMPONENT - Main', () => {
  it('renders correctly', () => {
    const component = create(<Main />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
