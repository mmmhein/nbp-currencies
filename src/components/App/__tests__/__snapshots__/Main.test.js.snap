// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Main renders correctly 1`] = `
.c0 {
  margin-top: 6rem;
}

<main
  className="c0"
/>
`;
